d3a_api_client/redis_device.py

```diff
@@ -1,15 +1,23 @@
 import logging
 import json
+import uuid
 from d3a_api_client.redis_client_base import RedisClient, Commands
+from d3a_interface.utils import wait_until_timeout_blocking
+
+
+class RedisAPIException(Exception):
+    pass
 
 
 class RedisDeviceClient(RedisClient):
     def __init__(self, device_id, autoregister=True, redis_url='redis://localhost:6379'):
         self.device_id = device_id
```

Comment:
We should change this to "device_name" or "device_slug" as it can be confusing with `self.device_uuid`
Also something for after the deploy